config/initializers/assets.rb

```diff
@@ -9,4 +9,6 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
 Rails.application.config.assets.precompile += %w(application_public.js)
+Rails.application.config.assets.precompile += %w( custom.css )
```

Comment:
It doesn't do anything. No error reported when custom.scss/custom.css is not there.